src/types.ts

```diff
@@ -9,7 +9,7 @@ export interface KintoRequest {
 
 export interface KintoRecord {
   id: string;
-  last_modified: number;
+  last_modified?: number;
```

Comment:
If the record comes from the server, it will always have this field. Which part of the tests required this?

Reply:
Particularly when updating a record, we know the ID, but we don't pass in a `last_modified` inside the record. I've added a new type, `KintoIdRecord`, which is used when we know a record's ID, but don't want to specify a `last_modified`. `KintoRecord` now extends from `KintoIdRecord`, adding the `last_modified` property.